sdk/core/core-util/test/public/datetime.spec.ts

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { toOffsetDateTime } from "../../src/datetime";
+import { assert } from "chai";
+
+describe("utcDateTime with timezone offset", function () {
+
+  describe("work in +08:00 timezone", function () {
+    it("converts a base64 string to bytes", function () {
+      const date = new Date("2022-01-05T19:08:10Z");
+      assert.strictEqual(toOffsetDateTime(date), "2022-01-05T19:08:10.000+08:00");
```

Comment:
@xirzec  I don't think the test case is correct for all timezones, but have no idea how to set a specific timezone, so that it works